### ISSUE TITLE
Order of installed versions when running `nvm list`

### ISSUE THREAD

**Issue by Sirikon** [coreybutler/nvm-windows#262]
### My Environment

- [ ] Windows 7 or below (not truly supported due to EOL - see wiki for details)
- [ ] Windows 8
- [ ] Windows 8.1
- [x] Windows 10
- [ ] Windows 10 IoT Core
- [ ] Windows Server 2012
- [ ] Windows Server 2012 R2
- [ ] Windows Server 2016

- [x] My Windows installation is non-English.

### I have already...

- [x] read the README to be aware of npm gotchas & antivirus issues.
- [x] reviewed the [wiki](https://github.com/coreybutler/nvm-windows/wiki) to make sure my issue hasn't already been resolved.
- [x] verified I'm using an account with administrative privileges.
- [x] searched the issues (open and closed) to make sure this isn't a duplicate.
- [x] made sure this isn't a question about how to use NVM for Windows, since [gitter](https://gitter.im/coreybutler/nvm-windows?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) is used for questions and comments.

### My issue is related to (check only those which apply):

- [ ] settings.txt
- [ ] proxy support
- [ ] 32 or 64 bit support

### Expected Behavior

When I use the command `nvm list` I expect to have the installed versions sorted from more recent to the past ones.

For example:

```
  * 6.10.1 (Currently using 64-bit executable)
    6.10.0
    6.9.2
    6.9.1
    6.8.0
    5.12.0
    4.2.3
```

### Actual Behavior

What I actually get is:

```
    6.9.2
    6.9.1
    6.8.0
  * 6.10.1 (Currently using 64-bit executable)
    6.10.0
    5.12.0
    4.2.3
```

### Steps to reproduce the problem:

1. Install, for example, versions `6.9.2`, `6.10.1` and `5.12.0`
2. Run `nvm list`

Seems like the versions are ordered alphabetically, and not with the SEMVER logic.

**Comment by coreybutler** [comment]
Does this always happen, or does the list change when you run `nvm ls` multiple times?

**Comment by Sirikon** [comment]
Seems to happen always, just tried and always got the same error.

**Comment by coreybutler** [comment]
OK - this is definitely strange. I've marked it for further exploration.

**Comment by TheJaredWilcurt** [comment]
Same thing for me:
```
    6.9.4
    6.5.0
    5.9.1
    4.4.5
    0.8.0
    0.7.0
    0.10.32
  * 0.10.12 (Currently using 64-bit executable)
```

**Comment by Sirikon** [comment]
Still having the issue: 

![image](https://cloud.githubusercontent.com/assets/1157382/26635878/5d3ef34e-461b-11e7-81b6-47b2c4a5a1a9.png)

Updated from version 1.1.3 just installing and overwritting.

**Comment by coreybutler** [comment]
I found the culprit and fixed it, as shown below. However; since I just released 1.1.5 a few hours ago (and since this is not a critical issue), I will wait until the next release to publish it.

![image](https://cloud.githubusercontent.com/assets/770982/26638099/e74e3480-45e6-11e7-9589-848673795d5b.png)

In the meantime, it is available in the master branch.

**Comment by Sirikon** [comment]
Cool! :)
